assets/prototype/application/ui/components/layout/editor-modal/useEditorModalState/types.ts

```diff
@@ -3,40 +3,38 @@
  */
 import { EntityId } from '../../../../../../domain/eventEditor/data/types';
 
-enum ActionType {
+export enum ActionType {
 	CLOSE_ALL = 'CLOSE_ALL',
 	CLOSE_EDITOR = 'CLOSE_EDITOR',
 	OPEN_EDITOR = 'OPEN_EDITOR',
 }
 
-interface ContextProps {
+export interface ContextProps {
 	editors: Editors | null;
 	editorState: EditorState | null;
 	id: EntityId | null;
 }
 
-type EditorAction = {
+export type EditorAction = {
 	editorId?: EditorId;
 	type: ActionType;
 };
 
-type EditorId = string;
+export type EditorId = string;
 
-interface Editors {
+export interface Editors {
 	addNewForm?: EditorId;
 	calculator?: EditorId;
 	editForm?: EditorId;
 	relations?: EditorId;
 }
 
-type EditorStack = EditorId[];
+export type EditorStack = EditorId[];
 
-interface EditorState {
+export interface EditorState {
 	closeAllEditors: () => void;
 	currentlyOpenEditor: () => EditorId | undefined;
 	isEditorOpen: (editorId: EditorId) => boolean;
 	closeEditor: (editorId: EditorId) => void;
 	openEditor: (editorId: EditorId) => void;
 }
-
-export { ActionType, ContextProps, EditorAction, EditorId, Editors, EditorStack, EditorState };
```

Comment:
Fix for this error:
![14_jan_8](https://user-images.githubusercontent.com/1477453/72363429-85583e00-36fd-11ea-8c87-63b7a5917c53.png)